README.md

```diff
@@ -1,8 +1,12 @@
-<img alt="Redox" height="90" src="https://github.com/redox-os/assets/raw/master/logo.png">
+<p align="center">
+<img alt="Redox" height="120" src="https://github.com/redox-os/assets/raw/master/logo.png">
+</p>
 
-**Redox** is an operating system written in pure Rust, designed to be secure and free. The website can be found at https://www.redox-os.org.
+**Redox** is an operating system written in Rust language, a language with focus on safety and high performance. Redox, following the microkernel design, aims to be secure, usable, and free. Redox is inspired by previous kernels and operating systems, such as SeL4, Minix, and BSD.
```

Comment:
It is redundant to say in Rust language, a language - it could be in Rust, a language